acro/acro.py

```diff
@@ -103,13 +104,18 @@ def __add_output(
         output : list[DataFrame]
             List of output DataFrames.
         """
-        name: str = f"output_{self.output_id}"
+
+        now = datetime.datetime.now()
+        timestamp = str(now.strftime("%Y-%m-%d-%H-%M-%S-%f")[:-4])
```

Comment:
If we remove some of the timestamp dashes we can shorten this and it will remove the warning about the sheet title length when writing excel. For example: "%Y-%m-%d-%H%M%S%f"